core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -948,7 +955,7 @@ private FileBlockInfo generateFileBlockInfo(InodeFile file, BlockInfo blockInfo)
       throws InvalidPathException {
     FileBlockInfo fileBlockInfo = new FileBlockInfo();
     fileBlockInfo.setBlockInfo(blockInfo);
-    fileBlockInfo.setUfsLocations(new ArrayList<WorkerNetAddress>());
+    fileBlockInfo.setUfsLocations(new ArrayList<String>());
```

Comment:
my guess is that this is intentional to avoid the overhead of computing the UFS locations if the file is in Alluxio